app/uk/gov/hmrc/debttransformationstub/utils/ifsrulesmasterspreadsheet/impl/IfsRulesMasterData.scala

```diff
@@ -85,6 +88,7 @@ final case class IfsRulesMasterData(
         case "SA into IFS and SoL AND NOT SSTTP" => RegimeUsage.`SA into IFS and SoL AND NOT SSTTP`
         case "SA SSTTP AND NOT into IFS and SoL" => RegimeUsage.`SA SSTTP AND NOT into IFS and SoL`
         case "SA SSTTP AND into IFS and SoL"     => RegimeUsage.`SA SSTTP AND into IFS and SoL`
+        case "SIA"                               => RegimeUsage.`Sia`
```

Comment:
The backticks ` are only needed when there are spaces. Use this instead:
```suggestion
        case "SIA"                               => RegimeUsage.Sia
```